openmetadata-ui/src/main/resources/ui/src/authentication/auth-provider/AuthProvider.tsx

```diff
@@ -122,13 +122,7 @@ export const AuthProvider = ({
 
   const handledVerifiedUser = () => {
     if (!isProtectedRoute(location.pathname)) {
-      const urlPathname = cookieStorage.getItem(REDIRECT_PATHNAME);
-      if (urlPathname) {
-        cookieStorage.removeItem(REDIRECT_PATHNAME);
-        history.push(urlPathname);
-      } else {
-        history.push(ROUTES.HOME);
-      }
+      history.push(ROUTES.HOME);
```

Comment:
@Ashish8689 , Can you confirm what we are trying to do here?

Reply:
@Sachin-chaurasiya After successful login user will be redirected to home page.